src/bigistore.c

```diff
@@ -1346,8 +1381,8 @@ is_bigistore_in_range(BigIStore *is, int64 *lower, int64 *upper, bool inclusive)
 static BigIStore *
 bigistore_limit(BigIStore *is, int64 *min, int64 *max)
 {
-    BigIStore      *result;
-    BigIStorePair  *pairs;
+    BigIStore *     result;
```

Comment:
Inconsistent formatting.

Reply:
yes unfortunately ClangFormat is unable to do that correctly :(